lib/plugins/aws/provider/awsProvider.js

```diff
@@ -233,6 +233,10 @@ class AwsProvider {
             UpdateReplacePolicy: { type: 'string' },
             Condition: { type: 'string' },
           },
+          awsLogGroupName: {
+            type: 'string',
+            pattern: '[.-_/#A-Za-z0-9]+',
```

Comment:
I believe we should either escape `-` or put it as last character, as otherwise it'll expand to range between `.` and `_` chars.

Also regex should be enclosed into `^` and `$`, as otherwise we confirm that any part in strings matches and not string as a whole